pkg/security/module/cws_others.go

```diff
@@ -10,14 +10,21 @@ package module
 import (
 	"github.com/DataDog/datadog-agent/pkg/eventmonitor"
 	"github.com/DataDog/datadog-agent/pkg/security/config"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
 func getFamilyAddress(config *config.RuntimeSecurityConfig) (string, string) {
 	return "tcp", config.SocketPath
 }
 
-// UpdateEventMonitorOpts adapt the event monitor options
-func UpdateEventMonitorOpts(opts *eventmonitor.Opts, config *config.Config) {}
+// UpdateEventMonitorOpts adapt the event monitor option
+func UpdateEventMonitorOpts(opts *eventmonitor.Opts, config *config.Config) {
+	if config.RuntimeSecurity.RemoteConfigurationEnabled {
+		log.Info("remote-configuration is not supported on this platform")
```

Comment:
nit: I would maybe add `CWS` in the log message so that customers understand where this is coming from

Reply:
good idea